src/modules/land_detector/MulticopterLandDetector.cpp

```diff
@@ -103,25 +96,24 @@ void MulticopterLandDetector::_update_topics()
 
 void MulticopterLandDetector::_update_params()
 {
-	param_get(_paramHandle.maxClimbRate, &_params.maxClimbRate);
-	param_get(_paramHandle.maxVelocity, &_params.maxVelocity);
-	param_get(_paramHandle.maxRotation, &_params.maxRotation_rad_s);
-	_params.maxRotation_rad_s = math::radians(_params.maxRotation_rad_s);
+	parameter_update_s param_update;
+
+	if (_param_update_sub.update(&param_update)) {
+		_update_params();
+		_freefall_hysteresis.set_hysteresis_time_from(false, (hrt_abstime)(1e6f * _param_lndmc_ffall_ttri.get()));
+		_param_lndmc_rot_max.set(math::radians(_param_lndmc_rot_max.get()));
```

Comment:
I suppose you can get away with this, but it seems kind of shady (remote possibility of accidentally committing the value). I would do the math::radians call at runtime to keep it simple.

Reply:
I agree. Honestly, I am a little bit in the dark about why this is happening in PX4:master.  What can we do better here?  I have checked this value in particular because it feels like it is converting radians as though they are degrees to radians again.  This one left me scratching my head.  I'll gladly change it to anything appropriate.

Reply:
Let me study what happens if I delete that LOC and report back...

Reply:
Well, my easiest tests are straightforward; flight testing is fine and the value reports back what it should after flights and reboots:
logs:
https://review.px4.io/plot_app?log=79c21d7b-908f-43dc-9aaa-6c0be1de1898
https://review.px4.io/plot_app?log=3c1b9939-5720-4373-b8cb-a5ee03ed19de

![image](https://user-images.githubusercontent.com/2497951/60378519-d2cec100-99e0-11e9-9c37-25520600c071.png)

Reply:
You're right, that was wrong. Fixed, rebased against current master and squashed into one commit.